### ISSUE TITLE
NPM 2.0> gives a warning when installing package

### ISSUE THREAD

**Issue by bebraw** [auth0/node-jsonwebtoken#47]
When installing the library using a recent NPM, it gives an error like

```
npm WARN engine jsonwebtoken@3.2.0: wanted: {"npm":"~1.4.28"} (current: {"node":"0.10.35","npm":"2.1.16"})
```

Maybe tweak `package.json` engines to get rid of this?

**Comment by jfromaniello** [comment]
Yes, sorry about that, fixed in version `3.2.1` commit 41cc7b7d8767322b39597172814349fbf88df6c2.

**Comment by bebraw** [comment]
Awesome, thanks.
